utils/githooks/pre-commit.d/30-Jenkinsfile.sh

```diff
@@ -19,6 +20,10 @@ if [ -z "$(_git_diff_cached_files "Jenkinsfile")" ] ; then
     exit 0
 fi
 
+echo "Temporary skipping as the old Jenkins instance is not accessible"
+echo "Pre-hook will be restored after new instance of Jenkins will be fully operational"
+exit 0
+
 echo "Checking syntax"
 
 HOST="${HOST:-build.hpdd.intel.com}"
```

Comment:
Isn't this covered by other PRs?

Reply:
No. That one is not ready yet.

Reply:
https://github.com/daos-stack/daos/pull/15995 has changes in this area.
Does not work for the other Jenkins as we have not got https:// access to it yet.

Reply:
It takes about 10s with #15995 before pre-hook detects that Intel's Jenkins is not available. This is why I propose to disable it fully.